conda-env/ci.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
     # =========================================
     - python >=3.10
     - setuptools
-    - cf_xarray >=0.9.1
```

Comment:
We should make the min version of `cf_xarray >=0.10.3` because that includes the fix for parametric coords: https://github.com/xarray-contrib/cf-xarray/releases/tag/v0.10.3

Reply:
Make sense wasn't sure what we were going to do about dropping 3.9 support, I see you answered that now. Thanks!